tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -0,0 +1,94 @@
+module.exports = {
+  commands: {
+    /**
+     *
+     * @param {string} fileName
+     */
+    openSharingDialog: function (fileName) {
+      const filesPage = this.api.page.filesPage()
+      const shareBtnSelector = filesPage.getFileRowSelectorByFileName(fileName) +
+        filesPage.elements['shareButtonInFileRow'].selector
+
+      this.initAjaxCounters()
+      filesPage.waitForFileVisible(fileName)
+        .useXpath()
+        .moveToElement(filesPage.getFileRowSelectorByFileName(fileName), 0, 0)
+        .click(shareBtnSelector)
+
+      return this.waitForElementVisible('@sharingSideBar')
+        .useCss()
+    },
+    /**
+     *
+     * @param {string} sharee
+     * @param {boolean} shareWithGroup
+     */
+    shareWithUserOrGroup: function (sharee, shareWithGroup = false) {
+      this.enterAutoComplete(sharee)
+        .waitForElementVisible('@sharingAutoCompleteDropDownElements')
+      return this.api
+        .initAjaxCounters()
+        .elements('css selector', this.elements['sharingAutoCompleteDropDownElements'].selector, (result) => {
```

Comment:
why not using `getShareAutocompleteItemsList`?

Reply:
Yes, we can do that. 
It requires some changes to `getShareAutoCompleteItemsList()` function
because currently it only returns users and groups name.